Tests/scripts/gitlab_slack_notifier.py

```diff
@@ -13,8 +13,9 @@
 DEMISTO_GREY_ICON = 'https://3xqz5p387rui1hjtdv1up7lw-wpengine.netdna-ssl.com/wp-content/' \
                     'uploads/2018/07/Demisto-Icon-Dark.png'
 ARTIFACTS_FOLDER = os.getenv('ARTIFACTS_FOLDER', './artifacts')
-ENV_RESULTS_PATH = os.getenv('ENV_RESULTS_PATH', os.path.join(ARTIFACTS_FOLDER, 'env_results.json'))
-PACK_RESULTS_PATH = os.path.join(ARTIFACTS_FOLDER, BucketUploadFlow.PACKS_RESULTS_FILE)
+ARTIFACTS_FOLDER_XSOAR = os.getenv('ARTIFACTS_FOLDER_XSOAR', './artifacts')
+ENV_RESULTS_PATH = os.getenv('ENV_RESULTS_PATH', os.path.join(ARTIFACTS_FOLDER_XSOAR, 'env_results.json'))
+PACK_RESULTS_PATH = os.path.join(ARTIFACTS_FOLDER_XSOAR, BucketUploadFlow.PACKS_RESULTS_FILE)
```

Comment:
Why `ARTIFACTS_FOLDER_XSOAR`? Won't it be the case that sometimes you'll be reporting on the mpv2 packs?

Reply:
we don't have that flow ready yet, so no